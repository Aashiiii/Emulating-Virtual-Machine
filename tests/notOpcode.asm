FUNC LABEL 0
    MOV REG 0 VAL 3
    MOV REG 1 VAL 0
    PRINT REG 0
    NOT REG 0
    PRINT REG 0
    NOT REG 1
    ADD REG 0 REG 1
    PRINT REG 0
    RET
