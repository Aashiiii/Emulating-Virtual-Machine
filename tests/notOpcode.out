3
252
251
